test/examples/lotr_migration.git/config

```diff
@@ -0,0 +1,8 @@
+[core]
+	repositoryformatversion = 0
+	filemode = true
+	bare = true
```

Comment:
I thought the script only works for non-bare repos?

Reply:
It does -- this test repo is cloned as non-bare during the test setup. But we can only include bare repos in `test/examples`.

Reply:
OK.